lib/xcode/install/simulators.rb

```diff
@@ -24,8 +24,8 @@ def run
     :private
 
     def install
-      filtered_simulators = @installed_xcodes.map(&:available_simulators).flatten.select do |sim|
-        sim.version.to_s.start_with?(@install)
+      filtered_simulators = @installed_xcodes.map(&:available_simulators).flatten.uniq{|x| x.name}.select do |sim|
```

Comment:
Oops, thank you. I should have had my editor find that for me first! I didn't yet understand what the `&:` construct was for.